### PR TITLE
Updates for 1.11.2 (OP-732)

### PR DESCRIPTION
**Pull request by mizzioisf**
Documentation 1.11.2 updates.

    Added documentation about the main menu log customization
    Small fixes

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -1931,22 +1949,18 @@ _log4j.properties_ as explained in the chapter <<log4j-properties,log4j.properti
...
 Minor releases (if not otherwise specified) do not need changes to the database structure, so the working copy can be replaced on each client with the new one and then connect to the same DB
 (see <<database-properties,database.properties>> and <<log4j-properties,log4j.properties>>).
 
 It is important to keep/preserve files that have been created or modified, especially configuration/settings and data files (see <<configuration,Configuration>>).
+=======
+Minor releases (if not otherwise specified) do not need changes to the database structure, so the working copy can be replaced on each client with the new one and then connect to the same DB
+(see <<database-properties,database.properties>> and <<log4j-properties,log4j.properties>>).
+
+It is important to keep/preserve files that have been created or modified, especially configuration/settings and data files (see <<config-files,Configuration Files>>).
+>>>>>>> develop
```
Comment (mwithi) [inline doc_admin/AdminManual.adoc]: this part looks like an unresolved conflict... 
the only difference I see is in the link `(see <<configuration,Configuration>>).` vs `(see <<config-files,Configuration Files>>).`

I think the former is the correct one, so the section between `=======` and `>>>>>>> develop` should be safe to remove

Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Yes, correct is  "<<configuration,Configuration>>"

Comment (mwithi) [inline doc_admin/AdminManual.adoc]:
```suggestion
```

**Review by mwithi — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**